package.json

```diff
@@ -26,33 +26,33 @@
   "dependencies": {
     "broccoli-funnel": "^1.1.0",
     "broccoli-merge-trees": "^2.0.0",
-    "ember-cli-babel": "^6.0.0",
+    "ember-cli-babel": "^6.8.0",
     "zxcvbn": "^4.3.0"
   },
   "devDependencies": {
     "broccoli-asset-rev": "^2.5.0",
-    "ember-cli": "~2.14.0-beta.2",
+    "ember-cli": "~2.16.2",
     "ember-cli-app-version": "^2.0.2",
-    "ember-cli-dependency-checker": "^1.3.0",
+    "ember-cli-dependency-checker": "^2.0.0",
+    "ember-cli-eslint": "^4.0.0",
```

Comment:
Thanks for fixing this package order!

Reply:
You're welcome!, but I can't take credit. Yarn did that!